src/WebJobs.Script.Host/Program.cs

```diff
@@ -19,11 +23,38 @@ static void Main(string[] args)
 
             ScriptHostConfiguration config = new ScriptHostConfiguration()
             {
-                RootScriptPath = rootPath
+                RootScriptPath = rootPath,
+                FileLoggingEnabled = true
```

Comment:
Ok. I thought this was test host. I'll remove. 

---
In reply to: [54119217](https://github.com/Azure/azure-webjobs-sdk-script/pull/26#discussion_r54119217) [](ancestors = 54119217)